src/bn_wallets.erl

```diff
@@ -201,7 +203,9 @@ handle_rpc(<<"wallet_pay_multi">>, {Param}) ->
                         Payee = ?jsonrpc_b58_to_bin(<<"payee">>, Entry),
                         Amount = ?jsonrpc_get_param(<<"bones">>, Entry, undefined),
                         Max = ?jsonrpc_get_param(<<"max">>, Entry, false),
-                        {Payee, Amount, Max}
+                        TokenBin = ?jsonrpc_get_param(<<"bones">>, Entry, <<"hnt">>),
```

Comment:
```suggestion
                        TokenBin = ?jsonrpc_get_param(<<"token">>, Entry, <<"hnt">>),
```

Reply:
Oh that's a good one!

Reply:
We were waiting for testnet to come back up; can probably test this now.